cmd/cli/lb.go

```diff
@@ -36,11 +36,10 @@ func StartLB(serverList string, port int) {
 			logger.LogInfo(fmt.Sprintf("[%s] %s\n", serverUrl.Host, e.Error()))
 			retries := handlers.GetRetryFromContext(request)
 			if retries < 3 {
-				select {
```

Comment:
For retries use a retry backoff, here a example: https://github.com/minio/minio-go/blob/master/retry.go

Reply:
I was reading this post  [backoff](https://blog.gopheracademy.com/advent-2014/backoff/)

```go
case <-time.After(backoff.Default.Duration(retries)):							
```